cinder/manifest.jsonnet

```diff
@@ -0,0 +1,193 @@
+local kpm = import "kpm.libjsonnet";
+
+function(
+  params={}
+)
+
+kpm.package({
+  package: {
+    name: "stackanetes/cinder",
+    expander: "jinja2",
+    author: "Quentin Machu",
+    version: "0.1.0",
+    description: "cinder",
+    license: "Apache 2.0",
+  },
+
+  variables: {
+    deployment: {
+      node_label: "openstack-control-plane",
```

Comment:
This label name should be moved to a central configutration file. So we don't need to change it in each component separetley.